cookbook/asset/custom_version_strategy.rst

```diff
@@ -0,0 +1,206 @@
+.. index::
+   single: Asset; Custom Version Strategy
+
+How to use a Custom Version Strategy for Assets
+===============================================
+
+Out of the box, Symfony exposes configuration options which allow you to
```

Comment:
I'd like to reword this paragraph. Instead of saying: "Symfony has some options to tweak things" ... we could say something like: "Symfony uses by default the XXX asset versioning strategy. Its behavior can be controlled with the xxx and yyy configuration options".